projects/izumi/index.js

```diff
@@ -35,8 +32,15 @@ Object.keys(chains).forEach(chain => {
 
 function getTvl(chain, isPool2) {
   return async (_, _b, { [chain]: block }) => {
-    const config = await getConfig(chains[chain]);
     var contracts
+    var config = {}
+
+    try {
+      config = await getConfig(chains[chain])
+    } catch {
```

Comment:
I dont like this pattern and we avoid it because, now we would get different tvls depending on whether the `getConfig` works. Can lead to spikes in tvl chart. Can we use static `configList` always for now and change to dynamic one once it is implemented on chain?

Reply:
Ok, we will put config to the chain asap.